packages/frontend/editor-ui/src/components/Folders/FolderBreadcrumbs.vue

```diff
@@ -43,15 +43,28 @@ const hiddenBreadcrumbsItemsAsync = ref<Promise<PathItem[]>>(new Promise(() => {
 // This will be used to filter out items that are already visible in the breadcrumbs
 const visibleIds = ref<Set<string>>(new Set());
 
-const currentProject = computed(
-	() => projectsStore.currentProject ?? projectsStore.personalProject,
-);
+// For shared context we do not show any folder breadcrumbs, only project badge
+const isSharedContext = computed(() => projectsStore.projectNavActiveId === 'shared');
```

Comment:
We have [a composable](https://github.com/n8n-io/n8n/blob/master/packages/frontend/editor-ui/src/composables/useProjectPages.ts) for detecting these. Can we use that here also?